src/main/scala/org/globalforestwatch/util/Util.scala

```diff
@@ -5,44 +5,41 @@ import java.io.{File, FileInputStream}
 import com.amazonaws.services.s3.AmazonS3URI
 import com.amazonaws.services.s3.model.ObjectMetadata
 import geotrellis.raster.RasterExtent
-import geotrellis.spark.SpatialKey
-import geotrellis.spark.io.index.zcurve.Z2
-import geotrellis.spark.io.s3.S3Client
-import geotrellis.spark.tiling.LayoutDefinition
+import geotrellis.layer.SpatialKey
```

Comment:
Codacy found an issue: [Use brackets to group imports from the same package](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)